### PR TITLE
fix(multicluster): add sections on enabling multicluster labels

### PR DESCRIPTION
**Pull request by douglas-reid**
Add sections on turning on `source|destination_cluster` labels for metrics in multicluster scenarios. 

[ X ] Policies and Telemetry

### CODE REVIEW

File: content/en/docs/setup/install/multicluster/gateways/index.md [hunk 1]
```diff
@@ -247,6 +247,287 @@ EOF
...
+destination cluster dimension.
+
+1. Copy the configuration for HTTP and TCP metrics to local disk.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system get envoyfilter stats-filter-1.6 -o yaml > stats-filter-1.6.yaml
+    $ kubectl -n istio-system get envoyfilter tcp-stats-filter-1.6 -o yaml > tcp-stats-filter-1.6.yaml
+    {{< /text >}}
+
+1. Replace the existing `spec` sections of **both** of those files, with the following content:
```
Comment (nmittler) [inline content/en/docs/setup/install/multicluster/gateways/index.md:266]: @douglas-reid this seems a bit heavy-handed for the install guide. Is there a way we can make this simpler? Could we ship a script with the samples, for example?

FYI @rcaballeromx

**Comment by douglas-reid** [comment]
Closing in favor of https://github.com/istio/istio/pull/23712

**Pull request closed without merging**